docs/custom-flows/user-impersonation.mdx

```diff
@@ -102,79 +102,121 @@ The above JSON snippet shows what a Clerk session token looks like when a user i
 
 When working with frontend (client-side) code, you can easily detect impersonated sessions and gain access to the actor ID or other information from the `act` claim.
```

Comment:
```suggestion
When working with client-side code, you can detect impersonated sessions and gain access to the actor ID or other information from the `act` claim.
```
I know you didn't change this line, but it's worth cleaning up a little for a quick win